### PR TITLE
Added sinhala e books for programming

### PR DESCRIPTION
**Pull request by Anuthra1997**

### CODE REVIEW

File: books/free-programming-books-si.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Index
+
+* [C++](#cpp)
+
+
+### <a id="cpp"></a>C++
+
+* [C++ පරිගණක ක්‍රමලේඛනය මුල  සිට  සරලව](http://www.mediafire.com/file/229c1v9n86bw2yf/C-Sinhala_Lanka_Learning.pdf/file) - කසුන් ලියනගේ
```
Comment (shrn01) [inline books/free-programming-books-si.md:8]: Please provide official links to the book

**Review by shrn01 — CHANGES_REQUESTED** [review]
Update the description

File: books/free-programming-books-si.md (hunk 1 above)
Comment (Anuthra1997) [reply books/free-programming-books-si.md]: The book is available on a local website for downloading (https://www.aluth.com/)
web post link: https://www.aluth.com/2018/06/c-learning-in-sinhala-pdf-book.html

**Comment by eshellman** [comment]
there's no pdf there, possibly a hostile site

File: books/free-programming-books-si.md (hunk 1 above)
Comment (Anuthra1997) [reply books/free-programming-books-si.md]: the download button is in sinhala language as  [මෙතනින් ] in the last line of the paragraph "එසේනම් [මෙතනින් ]පිවිස මෙම පොත බාගත කරගන්න."

**Comment by Anuthra1997** [comment]
the download button is in sinhala language as [මෙතනින් ] in the last line of the paragraph "එසේනම් [මෙතනින් ]පිවිස මෙම පොත බාගත කරගන්න."

**Pull request closed without merging**